### ISSUE TITLE
Question about new version

### ISSUE THREAD

**Issue by skelletR** [kdrag0n/safetynet-fix#242]
HI, the new version 2.4.0 when it will be released, it will be only zygisk? or will be and for riru etc? because i have magisk 23

**Comment by skelletR** [comment]
> Zygisk only. Why not update to Magisk 24?

I had updated,I have Xiaomi Poco X3 NFC with android 10  but I had some a battery issue and thermal and some lags, and I back to 23 is the perfect version for me, and because have the magiskhide feature, some say to stay to 23 because of this like HuskyDg

**Comment by kdrag0n** [comment]
If you're sure that's related, then I'd recommend reporting the issue to upstream Magisk. Zygisk is the way to go now and Riru is no longer maintained.

**Comment by skelletR** [comment]
> 

Understand, now I have magisk 23 with riru 26.1.3 and your module 2.1.1, maybe when you release the new version I try it again the new magisk

**Comment by skelletR** [comment]
> > Zygisk is the way to go now and Riru is no longer maintained.
> 
> It is because zygisk's LD_PRELOAD is difficult to hide itself than Riru's native bridge and many apps like Livin by Mandiri, Benefit pay, C6 Yellow,... are detecting for them

Exactly, that I mentioned, for that is one reason I stay to magisk 23

**Comment by skelletR** [comment]
https://github.com/HuskyDG, With the permission of https://github.com/kdrag0n, when he release the new version of SafetyNet, you can make it for work with magisk 23 with riru?

**Comment by skelletR** [comment]
> > https://github.com/HuskyDG, With the permission of https://github.com/kdrag0n, when he release the new version of SafetyNet, you can make it for work with magisk 23 with riru?
> 
> https://github.com/HuskyDG/safetynet-fix/tree/riru

Now must build with android studio? 🤔

**Comment by kdrag0n** [comment]
Android Studio is only used for editing code. Use `build.sh` to actually build the module.

**Comment by skelletR** [comment]
> Android Studio is only used for editing code. Use `build.sh` to actually build the module.

Oh, understand, thanks, I'm noob, must follow lot of instructions, the most can't understand, only understand need the git, for that I ask from huskyDg if he can can ported your 2.4.0 version for riru magisk 23

**Comment by skelletR** [comment]
https://github.com/HuskyDG, Thanks ported for magisk 23 with riru, only one question is the same exactly version 2.4.0 but on riru?
